src/test/store/__snapshots__/profile-view.test.js.snap

```diff
@@ -477,7 +477,7 @@ Object {
           null,
           Object {
             "URI": "https://mozilla.org",
-            "endTime": 6.5,
+            "endTime": 7,
```

Comment:
These times were updated in the previous PR too I think, and I still don't understand why 🤔

Reply:
Yeah, I think this was noise in the implementation changes, where I eventually got things correct. The final diff does not have them.